bucket/zeplin.json

```diff
@@ -0,0 +1,43 @@
+{
+    "homepage": "https://zeplin.io/",
+    "description": "Collaborative tool for designs' and styleguides' handoff with specs, assets and code snippets.",
+    "license": "MIT",
+    "version": "5.1.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/x64/Zeplin-5.1.1-full.nupkg",
+            "hash": "sha1:ada573adf43ce547c815706a1356d739227fef77"
+        },
+        "32bit": {
+            "url": "https://pkg.zeplin.io/windows/latestv2/ia32/Zeplin-5.1.1-full.nupkg",
+            "hash": "sha1:17a032825fa4d3914c948e47f8377f7dd55451b2"
+        }
+    },
+    "extract_dir": "lib\\net45",
+    "bin": "Zeplin.exe",
```

Comment:
What are available parameters?

Reply:
I've never used Zeplin via command line but `zeplin -h` returns this: 
```
Usage: Zeplin.exe [options]

Options:
  -V, --version             output the version number
  -e, --export <path>       export using dump at path
  -f, --squirrel-firstrun   squirrel-firstrun
  -i, --squirrel-install    squirrel-install
  -u, --squirrel-updated    squirrel-updated
  -r, --squirrel-uninstall  squirrel-uninstall
  -o, --squirrel-obsolete   squirrel-obsolete
  -h, --help                output usage information```

`--export` doesn't work on my computer, `-V` wrongly returns 1.0.0 and `squirrel` options are related to Zeplin's auto-updater.